pom.xml

```diff
@@ -160,7 +160,7 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
-            <version>1.5.11</version>
+            <version>1.7.30</version>
```

Comment:
Update required to match with logback, so that powermock tests can excute without failing due to class not found.